SAMtags.tex

```diff
@@ -530,10 +537,10 @@ \subsection{Base modifications}
 When multiple modifications are listed, for example {\tt C+mh,5,12,0;}, it indicates the modification may be any of the stated bases.
 The associated confidence values in the {\tt Ml} tag may be used to determine the relative likelihoods between the options.
 The example above is equivalent to {\tt C+m,5,12,0;C+h,5,12,0;}, although this will have a different ordering of confidence values in {\tt Ml}.
-Note ChEBI codes cannot be used in the multi-modification form (such as the {\tt C+mh} example above).
+Note ChEBI codes cannot be used in the multi-modification form (such as the {\tt C+.mh} example above).
```

Comment:
Is there a `C+.mh` example above?  We have (if fixed) a `C+?mh` one though.

Reply:
I'll fix this to `C+mh`, the `.` shouldn't have been there.